packages/cbioportal-clinical-timeline/src/Timeline.tsx

```diff
@@ -203,14 +250,31 @@ const Timeline: React.FunctionComponent<ITimelineProps> = observer(function({
         zoomSelectBox: useRef(null),
         zoomSelectBoxMask: useRef(null),
         cursorText: useRef(null),
+        hoverStyleTag: useRef(null),
+        timelineViewPort: useRef(null),
     };
 
+    const memoizedHoverCallback = useCallback(
+        (e: React.MouseEvent) => {
+            hoverCallback(e, refs.hoverStyleTag);
```

Comment:
I don't see the point of this memoization. e will always change, right? I could even see this being potentially problematic if react reuses the event objects

Reply:
it needs to be memoized because thats the only way to pass in refs.hoverStyleTag without using an anonymous function.   i actually think that we could probably get rid of the dependency, meaning this thing will only ever get created once.   i don't think identity of e matters.  it's just passed in every time and react shouldn't care if it's new or old or whatever.